initfiles/bash/etc/init.d/install-init.in

```diff
@@ -235,6 +235,21 @@ else
     log_success_msg
 fi
 
+if [ ! -d ${homePath}/certificate ]; then
+    mkdir -p ${homePath}/certificate
+fi
+
+if [ -d ${homePath}/certificate ] && [ ! -e ${homePath}/certificate/certificate.pem ]; then
```

Comment:
Not sure why [ -d ${homePath}/certificate ] here. If line 239 may not create the ${homePath}/certificate, should we stop or printf an error?